influxdb_connector/unittest/test_influxdb.py

```diff
@@ -0,0 +1,99 @@
+import os, sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import pandas as pd
+import unittest
+from influxdb_connector import Influxdb
+import yaml
+from yaml.loader import SafeLoader
+
+
+YAML_ARG_TO_TEST = "test_arguments"
+NUM_ROW_TO_TEST = 5
+
+
+class TestInfluxdb(unittest.TestCase):
+    def setUp(self) -> None:
+        cfg_path = os.path.dirname(os.path.abspath(__file__))
+        cfg_file = cfg_path + "/" + "test_config.yaml"
+        self.test_cfg = {}
+        with open(cfg_file) as c_info_file:
+            self.test_cfg = yaml.load(c_info_file, Loader=SafeLoader)
+        self.test_cfg = self.test_cfg[YAML_ARG_TO_TEST]
+        self.influxdb = Influxdb(
+            url=self.test_cfg["url"],
+            token=self.test_cfg["token"],
+            org=self.test_cfg["org"],
+        )
+
+
+@unittest.skip('skipping ')
```

Comment:
Wondering why skipping this important test case?
Also it would be nice to state the reason for skipping in the message. So other could be informed!

Reply:
Good catch, I had it uncommented while testing and forgot to comment it. It is now commented.